tests/test_urlslug.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+"""
+# Copyright (c) 2019, Rice University
+# This software is subject to the provisions of the GNU Affero General
+# Public License version 3 (AGPLv3).
+# See LICENCE.txt for details.
+"""
+import pytest
+
+from cnxcommon.urlslug import generate_slug
+
+def test_generate_slug():
+    expected_map = {
+        "College Physics / Viscosity and Laminar Flow; Poiseuille’s Law":
+        "/12-4/viscosity-and-laminar-flow-poiseulles-law",
```

Comment:
Hmm... I can see where this one is going to be interesting, because how would we know this chapter 12 section 4? The title isn't suggesting that at all. 🤔